heroic-core/src/main/java/com/spotify/heroic/metric/MetricManagerModule.java

```diff
@@ -402,11 +313,12 @@ public Builder(
             this.fetchParallelism = fetchParallelism;
             this.failOnLimits = failOnLimits;
             this.smallQueryThreshold = smallQueryThreshold;
-            this.mutateRpcTimeoutMs = mutateRpcTimeoutMs;
-            this.readRowsRpcTimeoutMs = readRowsRpcTimeoutMs;
-            this.shortRpcTimeoutMs = shortRpcTimeoutMs;
-            this.maxScanTimeoutRetries = maxScanTimeoutRetries;
-            this.maxElapsedBackoffMs = maxElapsedBackoffMs;
+
+            this.connectionSettings = Optional.of(new MetricsConnectionSettings(
+                    Optional.ofNullable(null),
```

Comment:
I'm guessing you want to keep this not configurable for time being?

Reply:
Yeah that looks odd, let me investigate.